documentation/asciidoc/accessories/camera/libcamera_hello.adoc

```diff
@@ -0,0 +1,66 @@
+
+The options are classified broadly into 3 groups, namely those that are common, those that are specific to still images, and those that are for video encoding. They are supported in an identical manner across all the applications where they apply.
+
+A complete list of all the command line options is given xref:camera.adoc#common-command-line-options[below].
+
+==== The Tuning File
+
+Raspberry Pi's `libcamera` implementation includes a _tuning file_ for each different type of camera module. This is a file that describes or "tunes" the parameters that will be passed to the algorithms and hardware to produce the best image quality. `libcamera` is only able to determine automatically the image sensor being used, not the module as a whole - even though the whole module affects the "tuning".
+
+For this reason it is sometimes necessary to override the default tuning file for a particular sensor.
+
+For example, the IMX378 sensor shows up as an IMX477 sensor even though it has different colour and vignetting responses, so if you have an IMX378 module you should use
+
+[,bash]
+----
+libcamera-hello --tuning-file /usr/local/share/libcamera/ipa/raspberrypi/imx378.json
+----
+
+If you are using a Soho Enterprises SE327M12 module you should use
+
+[,bash]
+----
+libcamera-hello --tuning-file /usr/local/share/libcamera/ipa/raspberrypi/se327m12.json
+----
+
+Notice how this also means that users can copy an existing tuning file and alter it according to their own preferences, so long as the `--tuning-file` parameter is pointed to the new version.
+
+Finally, the `--tuning-file` parameter, in common with other `libcamera-hello` command line options, applies identically across all the `libcamera-apps`.
+
+==== Preview Window
+
+Most of the `libcamera-apps` display a preview image in a window. When X Windows is not running it will draw directly to the display using Linux DRM (Direct Rendering Manager), otherwise it will attempt to use X Windows. Both paths use zero-copy buffer sharing with the GPU, and a consequence of this is that X forwarding is _not_ supported.
+
+The preview window can be suppressed entirely with the `-n` (`--nopreview`) option.
+
+The `--info-text` option allows the user to request that certain helpful image information is displayed on the window title bar using "% directives". For example
+
+[,bash]
+----
+libcamera-hello --info-text "red gain %rg, blue gain %bg"
+----
+will display the current red and blue gain values.
+
+For the HQ camera, use `--info-text "%focus"` to display the focus measure, which will be helpful for focusing the lens.
+
+A full description of the `--info-text` parameter is given xref:camera.adoc#common-command-line-options[below].
```

Comment:
Perhaps using "below" as link-text is unwise, in case the order of the sections gets shuffled about? :shrug: